audit/audit-gcp.sh

```diff
@@ -98,6 +98,31 @@ gcloud \
     | sed 's/.googleapis.com//' \
     | while read -r SVC; do
         case "${SVC}" in
+            bigquery)
+                mkdir -p "projects/${PROJECT}/services/${SVC}"
+                rm -f projects/${PROJECT}/services/${SVC}/bigquery.datasets.* || :
```

Comment:
Do you need this?  There's an "rm -rf" at the top of this script

Reply:
Done, removed